src/protocols/hpr_gwmp_router.erl

```diff
@@ -0,0 +1,112 @@
+-module(hpr_gwmp_router).
+
+-include("../grpc/autogen/server/packet_router_pb.hrl").
+
+-export([send/3]).
+
+-export([
+    packet_up_to_push_data/2,
+    route_to_dest/1,
+    txpk_to_packet_down/1
+]).
+
+-spec send(
+    Packet :: hpr_packet_up:packet(),
+    Stream :: grpcbox_stream:t(),
+    Routes :: hpr_route:route()
+) -> ok | {error, any()}.
+send(PacketUp, Stream, Route) ->
+    Hotspot = hpr_packet_up:hotspot(PacketUp),
+
+    case hpr_gwmp_udp_sup:maybe_start_worker(Hotspot, #{socket_dest => route_to_dest(Route)}) of
+        {error, Reason} ->
+            throw({gwmp_sup_err, Reason});
```

Comment:
What can we reasonably do here?
I think throwing will crash _all_ the senders, and we don't want that.

Reply:
`hpr_routing:handle_packet/2` can return {error, any()}.
If `hpr_routing:deliver_packet/3` can be changed to possibly return that, then `hpr_gwmp_router:send/3` can return that instead of `throw({gwmp_sup_err, Reason})`.

Would that work for us?